config/routes.rb

```diff
@@ -6,7 +6,7 @@
   devise_for :users, skip: [:registrations], controllers: { sessions: "sessions" }
 
   resources :users, only: [:edit, :update]
-  resources :profiles, except: [:show]
+  resources :profiles
```

Comment:
What is the reason for making profiles a singular resource?

I think it should remain plural - each user of the system has a profile.
Admin type users can CRUD profiles.